extensions/eo/README.md

```diff
@@ -78,6 +76,26 @@ numbers of several popular instruments.
 
 The difference between the `nir`, `nir08`, and `nir09` bands are that the `nir` band is a wider band that covers most of the spectral range of 0.75μm to 1.0μm. `nir08` and `nir09` are narrow bands centered 0.85μm and 0.95μm respectively. The same goes for the difference between `lwir`, `lwir11` and `lwir12`. 
 
+## Item fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| eo:gsd           | number                   | **REQUIRED.** Ground Sample Distance at the sensor. |
+| eo:cloud_cover   | number                   | Estimate of cloud cover as a percentage (0-100) of the entire scene. If not available the field should not be provided. |
+
+### Ground Sampling Distance
```

Comment:
Shouldn't the gsd be moved to core? Also, for Items just cloud_cover is left, that's not a lot actually and in principle there should be something required for Items, otherwise validation will basically break ;-)

Reply:
Hmmm, gsd must have made it's way back in there due to resolving merge conflicts.

Well, I suppose we could move cloud_cover also to the common Item metadata.

Reply:
It feels like cloud_cover is actually generic enough to be core, yes. Although... that would basically leave us with a "spectral bands extension" ;-)